apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -16,10 +22,11 @@ const PrimaryTest: React.FunctionComponent = () => {
   );
 };
 
-const PrimaryTestWithTitle: React.FunctionComponent = () => {
+const PrimaryTestWithTitleAndIcon: React.FunctionComponent = () => {
   return (
     <Notification
       variant={'primary'}
+      icon={{ svgSource: svgProps }}
```

Comment:
This creates a new object `{ svgSource: svgProps }` every time this is run, causing unnecessary renders. I'd prefer this is created on a separate line within the function then referenced. something like

```
const PrimaryTestWithTitleAndIcon: React.FunctionComponent = () => {
  const iconProps = { svgSource: svgProps };
  return (
    <Notification
       icon={{ svgSource: svgProps }}
       ...
  )

```